src/webview_provider/docsEditPanel.ts

```diff
@@ -249,6 +249,83 @@ export class DocsEditViewPanel implements WebviewViewProvider {
     await this.resolveWebviewView(this.panel!, this.context!, this.token!);
   };
 
+  private getTestDataByColumn(message: any, columnName: string) {
+    const tests = message.tests as undefined | TestMetaData[];
+
+    if (!tests?.length) {
+      this.terminal.debug(
+        "docsEditViewPanel:getTestDataByColumn",
+        "No test data passed",
+      );
+      return;
+    }
+
+    const columnTests = tests.filter((test) => test.column_name === columnName);
+
+    const filteredTests = columnTests
+      .map((test, i) => {
+        // If relationships test, set field and to
+        if (
+          test.test_metadata?.name === "relationships" &&
+          test.test_metadata.kwargs.field &&
+          test.test_metadata.kwargs.to
+        ) {
+          const { to, field } = test.test_metadata.kwargs;
+          return {
+            relationships: {
+              field,
+              to,
+            },
+          };
+        }
+
+        // set values if test is accepted_values
+        if (test.test_metadata?.name === "accepted_values") {
+          return {
+            accepted_values: { values: test.test_metadata.kwargs.values },
+          };
+        }
+        return test.test_metadata?.name;
+      })
```

Comment:
We need to test if there are any custom properties that we don't support and bail out in that case.